src/js/common/components/asyncLoader.jsx

```diff
@@ -0,0 +1,30 @@
+// Adapted from https://gist.github.com/acdlite/a68433004f9d6b4cbc83b5cc3990c194
+import React from 'react';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+
+export default function asyncLoader(getComponent, message) {
+  return class AsyncComponent extends React.Component {
+    static Component = null;
```

Comment:
It won't cause any problems because we're creating a new class each time this is used, so two uses of asyncLoader would never share the same class definition.

That said, I'm not entirely sure what the point of that is. I'll remove it.